manifest.yml

```diff
@@ -0,0 +1,9 @@
+---
+applications:
+    - name: opre-ops-test
+      buildpacks:
+        - https://github.com/cloudfoundry/python-buildpack.git#v1.7.44
```

Comment:
If we pin the precise version here, will we still get the benefit of cloud.gov sending us an email telling us to re-stage the application when a newer version of the buildpack is released? 

Looking at: https://cloud.gov/docs/getting-started/concepts/#buildpacks.

Reply:
Ah yeah, we can just bring in the `python_buildpack` if you think that's best, @alexsoble. I didn't know this was an option!